mta/widgetastic/__init__.py

```diff
@@ -43,6 +43,20 @@ def __init__(self, parent, locator, logger=None):
         self.locator = locator
 
 
+class MTADropdownMenu(Dropdown):
```

Comment:
There is `DropdownMenu` in the same class . Please check if it is not used at many place we can modify the same method or see if it can be reused.

Reply:
Updated existing widget(`DropdownMenu`) with new xapths